### PR TITLE
PKH-Flow - add Ausgaben subflow

### PR DESCRIPTION
**Pull request by rbrtrfl**
https://app.asana.com/0/1203259475859667/1207861176530990

### CODE REVIEW

File: app/flows/prozesskostenhilfeFormular/__test__/testcases.ts [hunk 1]
```diff
@@ -35,6 +35,9 @@ const cases = [
       "finanzielle-angaben/eigentum/kraftfahrzeuge-frage",
       "finanzielle-angaben/eigentum/gesamtwert",
       "finanzielle-angaben/eigentum-zusammenfassung/zusammenfassung",
+      "finanzielle-angaben/ausgaben/ausgaben-frage",
```
Comment (aaschlote) [inline app/flows/prozesskostenhilfeFormular/__test__/testcases.ts:38]: I see you have `hasAusgabenYes` guard but it doesn't reflect so much in this testcases, why?

File: app/flows/prozesskostenhilfeFormular/finanzielleAngaben/doneFunctions.ts [hunk 2]
```diff
@@ -58,3 +63,30 @@ export const eigentumDone: ProzesskostenhilfeFinanzielleAngabenGuard = ({
   context.hasWertsache !== undefined &&
   (!hasAnyEigentumExceptBankaccount({ context }) ||
     context.eigentumTotalWorth !== undefined);
+
+export const ausgabenDone: ProzesskostenhilfeFinanzielleAngabenGuard = ({
```
Comment (aaschlote) [inline app/flows/prozesskostenhilfeFormular/finanzielleAngaben/doneFunctions.ts:67]: do we have unit test for that?

File: app/flows/prozesskostenhilfeFormular/finanzielleAngaben/flow.json [hunk 3]
```diff
@@ -538,6 +538,177 @@
           }
         }
       }
+    },
+    "ausgaben": {
```
Comment (aaschlote) [inline app/flows/prozesskostenhilfeFormular/finanzielleAngaben/flow.json:542]: I don't see unit test for this cases here. Is it expected?

**Review by aaschlote — COMMENTED** [review]
Please take a look the if the unit tests are needed for these changes!

File: app/flows/prozesskostenhilfeFormular/__test__/testcases.ts (hunk 1 above)
Comment (rbrtrfl) [reply app/flows/prozesskostenhilfeFormular/__test__/testcases.ts:38]: sorry what do you mean by that?

**Pull request merged**